cirq/google/engine/engine.py

```diff
@@ -619,6 +619,103 @@ def remove_job_labels(self, job_resource_name: str, label_keys: List[str]):
             fingerprint = job.get('labelFingerprint', '')
             self._set_job_labels(job_resource_name, new_labels, fingerprint)
 
+    def list_processors(self, project_id: str) -> List[Dict]:
+        """ Returns a list of Processors that the user has visibility to in the
```

Comment:
Nit, alignment of doc string should be against the """ and should just word wrap.  

Similar in other methods below.